src/components/Catalog.vue

```diff
@@ -397,15 +408,21 @@ export default {
           } else {
             this.externalItemCount = items.features.length;
           }
+          Object.keys(this.externalPaginationLinks).forEach(linkRelation => {
+            const link = items.links.find(link => link && link.rel === linkRelation && typeof link.href === 'string');
+            if (link && typeof link.href === 'string') {
+              this.externalPaginationLinks[linkRelation] = `${this.slugify(link.href.replace(this.ancestors[0], '').replace('/items', ''))}`;
```

Comment:
```suggestion
              this.externalPaginationLinks[linkRelation] = `${this.slugify(link.href)}`;
```

I think this is enough and should solve issues with EODC etc. But this needs to be a query string or so...

Reply:
It works without `.replace(this.ancestors[0], '')` but I still need `.replace('/items', '')` otherwise the `/items` is carried into the slug and it fails to load data later on (something with the hash being not detected or absent).
Should I look to fix `common.js` in this regards instead of removing the `/items` part?

Reply:
Removing the items leads to issues with other implementations as mentioned above. Unfortunately, I hadn't had the chance yet to look into it myself and as I've only recently taken over maintenance of this project, I'm not yet 100% familiar with how the routing works myself, but on first sight it looks like the replace needs to be removed and we need to look into it why it does not work then for some cases.

Reply:
So I've add some time to look this into a little more, from what I can tell I'm stuck.
Either I remove `/items` part of my (external) pagination links so that the hash contains `/collections/${CatalogueID}?param1=...&param2=...` or I leave it as is (so something like `/collections/${CatalogueID}/items?param1=...&param2=...` but then it fails both ways.
I also tried to do something like `${HashB58Catalogue}/${HashB58Pagination}/` but that doesn't work either, because at some point it loads the list (`FeatureCollection`) as if it is the catalogue metadata and fails. So we have to remove this `/items` part somewhere from what I can tell. Any idea?

Reply:
I don't know if what I've just published is a better alternative, but at least I don't replace arbitrary part of the URL anymore (so if someone has `/items/` somewhere in its backend URL it should also work...)
I'm using the URL given as prop, adding any URL param from the pagination link, and slugying this as the new path.

Reply:
I'll look at it after my vacation, thanks for all the effort.

Reply:
Sorry for the slow reply, it was pretty busy the last time. 
I've checked several catalogs again. So most catalogs seem to work, but the EODC issue still exists. I still have to figure out where the "/items" gets lost...

Reply:
Okay, I dig through the code and found numerous issues. Unfortunately, it is very hard to get it right with the current architecture of STAC Browser 1.0 and this implementation will always be a temporary "workaround". We need to get that right in 2.0.

1. The first page of the items is requested 3(!) times: 
![grafik](https://user-images.githubusercontent.com/8262166/114301892-f3d5a880-9ac6-11eb-979d-2bb5798f0ae0.png)
2. Interestingly, the computed next link for the first request is correct. The latter responses are made into "incorrect" next links.
3. A possible cause could be that you assign new values to "this" properties in a computed property. That is a side-effect, which is unexpected and may lead to a recomputation of the computed property itself. Not sure whether it's the actual cause, but that needs to be fixed. A watcher on externalItems is likely the way to go. So everything that is below `const items = await rsp.json();` in the async comp. property `externalItems` needs to likely go into a watcher for the `externalItems` property. I later realized that there is a circular dependency between `externalItemsURL` and `hasExternalPagination`, i.e. the results of those computed properties depend on each other. I had to remove the code related to hasExternalPagination from the externalItemsURL prop.

It also seems that the EODC API has a bug so that the first next link leads to page 1 again.